src/azul/service/repository_controller.py

```diff
@@ -263,7 +266,7 @@ def download_file(self,
                     pass
                 elif wait == '1':
                     time_slept = self.server_side_sleep(float(retry_after))
-                    retry_after = round(retry_after - time_slept)
+                    retry_after = ceil(retry_after - time_slept)
```

Comment:
The change from `round` to `ceil` is currently in a commit labeled "Add a default value for the /repository/files wait parameter". Please explain why going from `round` to `ceil` is related to adding a default or isolate that change in its own commit.